gulpfile.babel.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+// This gulpfile makes use of new JavaScript features.
+// Babel handles this without us having to do anything. It just works.
+// You can read more about the new JavaScript features here:
+// https://babeljs.io/docs/learn-es2015/
+
+import path from 'path';
+import gulp from 'gulp';
+import del from 'del';
+import runSequence from 'run-sequence';
+// import swPrecache from 'sw-precache';
+import gulpLoadPlugins from 'gulp-load-plugins';
+// import pkg from './package.json';
+import merge from 'merge-stream';
+
+const $ = gulpLoadPlugins();
+
+// Compile and automatically prefix stylesheets
+gulp.task('styles', () => {
+  const AUTOPREFIXER_BROWSERS = [
+    'ie >= 10',
+    'ie_mob >= 10',
+    'ff >= 30',
+    'chrome >= 34',
+    'safari >= 7',
+    'opera >= 23',
+    'ios >= 7',
+    'android >= 4.4',
+    'bb >= 10'
+  ];
+
+  // For best performance, don't add Sass partials to `gulp.src`
+  return gulp.src([
+    'static/sass/**/*.scss'
+  ])
+    .pipe($.sass({
+      outputStyle: 'compressed',
+      precision: 10
+    }).on('error', $.sass.logError))
+//    .pipe($.autoprefixer(AUTOPREFIXER_BROWSERS))
```

Comment:
As in my other comment, I was leaving this for #319, and it simplified my testing.